plugins/plugin-site/src/components/PluginWikiContent.jsx

```diff
@@ -1,9 +1,46 @@
-import React from 'react';
+import React, {useState} from 'react';
 import PropTypes from 'prop-types';
+import {lightbox} from './PluginWikiContent.module.css';
 
 const PluginWikiContent = ({wiki}) => {
+    const [lightboxImage, setLightboxImage] = useState(null);
+
+    const handleClick = (event) => {
+        // Find the parent <a> tag with target="_blank"
+        let parentNode = event.target.parentNode;
+
+        while (parentNode && parentNode.tagName !== 'A') {
+            parentNode = parentNode.parentNode;
+        }
+
+        if (
+            parentNode &&
+            parentNode.tagName === 'A' &&
+            parentNode.getAttribute('target') === '_blank'
+        ) {
+            // Prevent the default behavior of the anchor tag (prevents redirection to github page)
```

Comment:
```suggestion
            // Prevent the default behavior of the anchor tag from redirecting to the GitHub page
```

Reply:
@sridamul Is this what you originally meant? Would it be better to rephrase for clarity?

Reply:
>  Is this what you originally meant? Yes